### DISCUSSION TITLE
官方文档里印章检测md中提及AI studio项目未公开，无法运行

### DISCUSSION THREAD

**Discussion by yubuyuabc** [PaddlePaddle/PaddleOCR#12349] [Q&A]
(https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/%E5%8D%B0%E7%AB%A0%E5%BC%AF%E6%9B%B2%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB.md)

**Comment by TingquanGao** [comment]
收到，我们确认一下尽快修改。谢谢~

**Comment by TingquanGao** [comment]
https://aistudio.baidu.com/education/teacher/management/26849 公开了，加入课程后，在课节6中可以看到：

<img width="852" alt="截屏2024-02-28 14 31 23" src="https://github.com/PaddlePaddle/PaddleOCR/assets/12560511/b5f7c089-821d-4269-9c8a-72deb00cc5e1">

**Comment by yubuyuabc** [comment]
> https://aistudio.baidu.com/education/teacher/management/26849 公开了，加入课程后，在课节6中可以看到：
> 
> <img alt="截屏2024-02-28 14 31 23" width="852" src="https://private-user-images.githubusercontent.com/12560511/308444432-b5f7c089-821d-4269-9c8a-72deb00cc5e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkxMTA0NTAsIm5iZiI6MTcwOTExMDE1MCwicGF0aCI6Ii8xMjU2MDUxMS8zMDg0NDQ0MzItYjVmN2MwODktODIxZC00MjY5LTljOGEtNzJkZWIwMGNjNWUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI4VDA4NDkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4Y2YzYzVjNGNkZDIyOTNmNTc1ZjQ4NWY5ZTc3MGFiNDRlMmNiMTUwYjYwNjIwZjZhMzdmYTA3YjM1ZTU1MDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fSd0A62uusfFTl43-RNHPJDwmEDrEEaWgzmhvA1StRs">

该课程存在一些配置文件缺失的问题，麻烦确认下是否是版本问题
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/58143197/a8566963-7b78-4e66-bf74-be47ddf3d37f)
